docs/application/dotnet/api/netstandard-limitations.md

```diff
@@ -1,49 +1,47 @@
-Even these event handlers are not called when the stream is closed.
+    In case of Process.OutputDataReceived and Process.ErrorDataReceived, an event occurs but null is passed as DataReceivedEventArgs.Data. Even these event handlers are not called when the stream is closed.
 
 
-## Access to process components for other process resources is not allowed
+2. Access to process components for other process resources is not allowed
 
-The APIs, which get other process components such as Process.GetProcesses() and Process.GetProcessesByName(), throw an exception because an application is not allowed to get information about other processes.
+    The APIs, which get other process components such as `Process.GetProcesses()` and `Process.GetProcessesByName()`, throw an exception because an application is not allowed to get information about other processes.
 
-In addition, GetProcessById() with other process ID also throws an exception.
+    In addition, `Process.GetProcessById()` with other process ID also throws an exception.
 
 
-## Limited System.IO.MemoryMappedFile support
+3. Limited System.IO.MemoryMappedFile support
 
-Creating MemoryMappedFile with ***execute*** access capability is not supported. So MemoryMappedFile.CreateViewAccessor() and MemoryMappedFile.CreateViewStream() with both MemoryMappedFileAccess.ReadExecute and MemoryMappedFileAccess.ReadWriteExecute throw an exception.
+    Creating MemoryMappedFile with **execute** access capability is not supported. So `MemoryMappedFile.CreateViewAccessor()` and `MemoryMappedFile.CreateViewStream()` with both MemoryMappedFileAccess.ReadExecute and MemoryMappedFileAccess.ReadWriteExecute throw an exception.
 
-In addition, MemoryMappedFile.CreateNew() and MemoryMappedFile.OpenExisting() are not supported because named maps are not supported.
+    In addition, `MemoryMappedFile.CreateNew()` and `MemoryMappedFile.OpenExisting()` are not supported because named maps are not supported.
 
 
-## System.Net.NetworkInformation.Ping is not supported
+4. System.Net.NetworkInformation.Ping is not supported
 
-Sending an Internet Control Message Protocol (ICMP) echo message by System.Net.NetworkInformation.Ping is not supported.
+    Sending an Internet Control Message Protocol (ICMP) echo message by System.Net.NetworkInformation.Ping is not supported.
 
 
-## Limited system special folders support
+5. Limited system special folders support
 
-Only following system special folders are supported on Tizen.
-- LocalApplicationData
-- MyMusic
-- MyPictures
-- MyVideos
+    The following system special folders are supported on Tizen.
+    - LocalApplicationData
+    - MyMusic
+    - MyPictures
+    - MyVideos
 
-Your application needs http://tizen.org/privilege/mediastorage privilege to access them, except LocalApplicationData, with r/w access mode.
+    Your application needs http://tizen.org/privilege/mediastorage privilege to access them, except LocalApplicationData, with **read** or **write** access mode.
```

Comment:
access the special folders.  Except LocalApplicationData with **read** or **write** access mode.